core/scheduler.py

```diff
@@ -116,7 +116,7 @@ def scan_directory():
             if r:
                 logging.info('Found match for {} in releases: {}.'.format(fname, r['title']))
             else:
-                r['guid'] = 'POSTPROCESSING{}'.format(b16encode(fname.encode('ascii', errors='ignore')).decode('utf-8').zfill(16)[:16])
+                r['guid'] = 'postprocessing{}'.format(hashlib.md5(fname.encode('ascii', errors='ignore')).hexdigest())
```

Comment:
I assumed b16 would be faster but after a quick test it looks like md5 is slightly faster. Both call C functions for the actual work but I suppose the md5 algorithm is so optimized at this point in time.

Reply:
I understand the focus on performance but there is also a need to be correct: very frustrating to play a movie that is not actually what its filename tells it is.
Furthermore as mover/renamer overwrite existing files without warning it can mess the current library.

As b16encode do not apply any rotate/swap of bytes then it is not a good hash function.

Reply:
The purpose is simply to have a unique string in the database as a fake guid. Using md5 or b16 doesn't make any difference in how the string is then used since an identical directory or filename will result in an identical guid. There is no real need to obscure the data at all, I simply wanted to have a guid-like representation.

Changing this to md5 will have no affect on anything other than performance. Which is actually slightly better than b16 for whatever reason.